.github/pull_request_template.md

```diff
@@ -15,4 +15,14 @@ Please check you have done the following:
 - [ ] Added any new containers to `.github/workflows/docker-build-push.yml` and `.github/workflows/docker-linting.yml`
 - [ ] Added any new workflows to `.github/workflows/workflow-testing.yml`
 - [ ] Added any new test data to `test_data`
-- [ ] When changing an existing module, have you bumped the version? Major.Minor.Patch - Major for breaking changes, Minor for feature updates and most things, Patch for bug fixes and small things like comment and variable name updates
+- [ ] Bumped the version when changing an existing module
+  - Major.Minor.Patch
+    - Major for breaking changes
+    - Minor for feature updates and most things
+    - Patch for bug fixes and small things like comment and variable name updates
+
+## Review
+
+- [ ] Selected some reviewers
+- [ ] Assigned yourself to the PR
+- [ ] Tagged the PR appropriates
```

Comment:
This looks good! Just a small question: should it be "appropriately" instead of "appropriates"? @amchakra

Reply:
Yes indeed! Corrected